base/shared/src/main/scala/scalaz/tc/foldable.scala

```diff
@@ -16,6 +17,12 @@ trait FoldableClass[F[_]] {
   def msuml[A](fa: F[A])(implicit A: Monoid[A]): A =
     foldLeft(fa, A.mempty)(A.mappend(_, _))
 
+  def traverse_[T[_], A, B](ta: F[A])(f: A => T[B])(implicit T: Applicative[T]): T[Unit] =
+    foldLeft(ta, T.pure(()))((_, a) => T.map(f(a))(_ => ()))
```

Comment:
@hrhino I'd like to do `foldLeft(ta, T.pure(()))((r, a) => r <* f(a))`. 
What should I import to get `<*`?